Modules/System/Time Zone Management/app.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id": "4c315253-b024-4fdc-9c3d-794036768af5",
+  "name": "Time Zone Management",
```

Comment:
I wonder if we really need a module for "Time Zone Management", or if it maybe would make sense to make the module a tad wider? We've got a lot of DateTime conversion functions in the BaseApp - maybe these could find a new home in this module too?

Reply:
@JesperSchulz What are thinking of?
Something like "Date and Time" or "Date & Time Utility", ...
And if this becomes whole Date,DateTime,DateTimeOffset,TimeZone Module. Are the Id's still the correct ones or should they be changed to some where we can easily add several more objects without the need for additional Id Ranges?

Reply:
Exactly, a "Date & Time" module. That'd be lovely. And no, we would not have to change the ID's. While ranges are preferable, they're not a requirement. We can pick individual IDs.

Reply:
Since we are having two similar conversations on this topic, I am marking this one as resolved and we can add any details on the one below this.